example/cavity/Fluid/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd ${0%/*} || exit 1
+. $WM_PROJECT_DIR/bin/tools/RunFunctions
+
+restore0Dir
+touch res.foam
```

Comment:
Todo when we move this case to the tutorials:

- Move the files from `0.orig` to just `0` (unless there is a good reason for that, I hope I am not overseeing a previous comment here).
- Restructure the tutorial files/directories similarly to the tutorials repository: `cavity/fluid-openfoam` and `cavity/solid-mbdyn`.
- Rename the `res.foam` to `fluid-openfoam.foam`.